app/src/main/java/com/segment/analytics/android/integrations/appsflyer/AppsflyerIntegration.java

```diff
@@ -65,6 +65,15 @@ public class AppsflyerIntegration extends Integration<AppsFlyerLib> {
     private String customerUserId;
     private String currencyCode;
 
+    public static void setManualMode(Boolean manualMode) {
+        AppsflyerIntegration.manualMode = manualMode;
+    }
+
+    public static void startAppsFlyer(Context context){
+        AppsFlyerLib.getInstance().start(context);
```

Comment:
Should we check if Context is null?
If yes, should we print something in the debug logs?